app/models/case_contact.rb

```diff
@@ -102,7 +102,7 @@ class CaseContact < ApplicationRecord
     when /^occurred_at/
       order(occurred_at: direction)
     when /^contact_type/
-      joins(:contact_types).order(name: direction)
+      joins(:contact_types).merge(ContactType.order(name: direction))
```

Comment:
This is a little fancier than the simpler `order("contact_types.name #{direction}")`, but ensures we're using everything ActiveRecord gives us when building the query.